__tests__/app.test.js

```diff
@@ -29,6 +29,70 @@ describe("/api/", () => {
     });
   });
   describe("/api/reviews", () => {
+    describe('"/api/reviews', () => {
+      test("200: responds with all reviews if no query is set in descending order", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then((reviews) => {
+            const body = reviews.body;
+
```

Comment:
data from your api should be sent back in an object with a key representing what data is being sent back

```diff
-   .then((reviews) => {
-            const body = reviews.body;
+   .then((response) => {
+            const reviews = response.body.reviews;
```